src/wip/fixed-trader.rain

```diff
@@ -1,11 +1,11 @@
-# Strategy: Fixed Grid
+# Strategy: Fixed Trader
 #
 # High level, this strategy trades mean amount of BLUE denominated RED tokens at the end of every cooldown period
```

Comment:
do we really want to be using a cooldown or would a per second budget work better?

for example, this strat has no DOS protection against someone dusting the cooldown, but a per second budget doesn't have this issue